apps/web/public/static/locales/en/common.json

```diff
@@ -1451,6 +1467,11 @@
   "duration_limit_reached": "Duration Limit for this event type has been reached",
   "admin_has_disabled": "An admin has disabled {{appName}}",
   "disabled_app_affects_event_type": "An admin has disabled {{appName}} which affects your event type {{eventType}}",
+  "event_replaced_notice": "An admin has replaced one of your event types",
+  "email_subject_slug_replacement": "A team administrator has replaced your event /{{slug}}",
+  "email_body_slug_replacement_notice": "An administrator on the <strong>{{teamName}}</strong> team has replaced your event type <strong>/{{slug}}</strong> with a managed event type that they control.",
+  "email_body_slug_replacement_info": "Your link will continue to work but somesettings for it may have changed. You can review it in event types.",
```

Comment:
```suggestion
  "email_body_slug_replacement_info": "Your link will continue to work but some settings for it may have changed. You can review it in event types.",
```